packages/shopify-codemod/transforms/utils.js

```diff
@@ -11,3 +11,14 @@ export function findLastMember(node) {
   }
   return node;
 }
+
+export function insertAfterDirectives(body, newNode) {
+  let i = 0;
+  for (;i < body.length; i++) {
```

Comment:
As shit, yeah. Gross :( could do it with a weird `findIndex` but whatever, probably better this way.